proxy/IPAllow.cc

```diff
@@ -153,6 +154,46 @@ IpAllow::Print()
       }
     }
   }
+  for (IpMap::iterator spot(_dest_map.begin()), limit(_dest_map.end()); spot != limit; ++spot) {
```

Comment:
Quinn will change this to be two calls to the same method, passing in the two maps.